physionet-django/project/serializers.py

```diff
@@ -0,0 +1,16 @@
+# from user.models import User
+from rest_framework import serializers
+
+from project.models import PublishedProject
+
+
+class PublishedProjectSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = PublishedProject
+        fields = ('id', 'title', 'abstract')
+
+
+class PublishedProjectDetailSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = PublishedProject
+        fields = '__all__'
```

Comment:
I think this is a little dangerous. We should be explicit about what fields on the model are being exposed by the API.

Reply:
Update fields as requested by Alistar

Reply:
Changed as per Alistair's suggestion